test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1086,6 +1087,104 @@ func Test_RunPodSandbox_MultipleContainersSameVhd_LCOW(t *testing.T) {
 	}
 }
 
+func Test_RunPodSandbox_MultipleContainersSameVhd_WCOW(t *testing.T) {
+	pullRequiredImages(t, []string{imageWindowsNanoserver})
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	annotations := map[string]string{
+		"io.microsoft.virtualmachine.computetopology.memory.allowovercommit": "true",
+	}
+
+	vhdHostDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %s", err)
+	}
+	defer os.RemoveAll(vhdHostDir)
+
+	vhdHostPath := filepath.Join(vhdHostDir, "temp.vhdx")
+
+	if err = hcs.CreateNTFSVHD(ctx, vhdHostPath, 10); err != nil {
+		t.Fatalf("failed to create NTFS VHD: %s", err)
+	}
+
+	vhdContainerPath := "C:\\containerDir"
+
+	mounts := []*runtime.Mount{
+		{
+			HostPath:      "vhd://" + vhdHostPath,
```

Comment:
Have you tried running this test? It looks like this will fail due to `vhdHostPath` not being properly escaped.